rollouts/controllers/rollout_controller.go

```diff
@@ -453,8 +458,9 @@ func (r *RolloutReconciler) computeTargets(ctx context.Context,
 
 func pkgNeedsUpdate(rollout *gitopsv1alpha1.Rollout, rrs gitopsv1alpha1.RemoteRootSync, pkg *packagediscovery.DiscoveredPackage) (*gitopsv1alpha1.RemoteRootSync, bool) {
 	// TODO: We need to check other things here besides git.Revision and metadata
+	metadata := getSpecMetadata(rollout)
```

Comment:
I would consider creating a pull request with just this change since this change is technically unrelated to the rest of the changes in the pull request.

Reply:
Thanks for pointing it out. This will go away once @natasha41575 PR (that has the fix) is merged.